browser/PostMessageTransport.js

```diff
@@ -0,0 +1,100 @@
+/*jslint vars: true, plusplus: true, devel: true, nomen: true, indent: 4, forin: true, maxerr: 50, regexp: true */
+/*global define */
+
+// This transport provides a PostMessage connection between Brackets and a live browser preview.
+
+define(function (require, exports, module) {
+    "use strict";
+
+    var _iframeRef;
+    
+    var EventDispatcher = require("utils/EventDispatcher");
+    
+    // The script that will be injected into the previewed HTML to handle the other side of the post message connection.
+    var PostMessageTransportRemote = require("text!PostMessageTransportRemote.js");
+
+    EventDispatcher.makeEventDispatcher(exports);
+    
+    /**
+     * Saves a reference of the iframe element to a local variable
+     * @param {jQuery reference}
+     */
+    function setIframe($ref){
+        if($ref)
+            _iframeRef = $ref;
+    }
+
+    /**
+    * Initializes the post message connection
+    */
+    function start(){
+        var win = _iframeRef.get(0).contentWindow;
```

Comment:
This can just be `_iframeRef.contentWindow;`, since we won't pass a jQuery object.